#!/bin/sh

for i in dev-libs/libbsd/libbsd-*.ebuild ; do
	cat > bsd.patch << EOF
--- a/${i}
+++ b/${i}
@@ -14,6 +14,6 @@
 IUSE="static-libs"
 
-DEPEND=">=sys-kernel/linux-headers-3.17"
+DEPEND="kernel_linux? ( >=sys-kernel/linux-headers-3.17 )"
 RDEPEND=""
 
 pkg_setup() {
EOF
	git apply bsd.patch || exit 1
	rm -f bsd.patch || exit 1
done
